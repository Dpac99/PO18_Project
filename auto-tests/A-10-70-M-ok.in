100013
1
e1.obj
5
2
D1
P4
3
4
1
D1
P5
234
0
2
0
