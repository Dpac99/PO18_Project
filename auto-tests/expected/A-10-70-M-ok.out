Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Ficheiro a abrir: Pode preencher inquérito do projecto P1 da disciplina D1
Pode preencher inquérito do projecto P2 da disciplina D1
Pode preencher inquérito do projecto P4 da disciplina D1
Resultados do inquérito do projecto P1 da disciplina D1
Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Número de horas de trabalho: Comentário ao projecto: Preencher inquérito: Operação inválida: O projecto 'P4' não existe na disciplina 'D1'.
Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: 